packages/lib/auth.ts

```diff
@@ -54,7 +54,14 @@ export function isPasswordValid(password: string, breakdown?: boolean, strict?:
       if (password[i] === password[i].toLowerCase()) low = true;
     }
   }
-  return !!breakdown ? { caplow: cap && low, num, min, admin_min } : cap && low && num && min;
```

Comment:
We were actually always returning the `admin_min` boolean, even when we shouldn't check for it. This caused the boolean to be always false when strict was disabled.